#! /bin/sh /usr/share/dpatch/dpatch-run
## 99-unnamed.dpatch by  <abi@debian.org>
## fix login problem if http://localhost/ is used.
## see: http://www.zabbix.com/forum/showthread.php?t=4033

@DPATCH@
--- data.sql.deb	2006-09-20 14:36:43.000000000 +0200
+++ zabbix-1.1.2/create/data/data.sql	2006-09-20 14:37:01.000000000 +0200
@@ -33,8 +33,8 @@
 -- Dumping data for table `users`
 --
 
-INSERT INTO users VALUES (1,'Admin','Zabbix','Administrator','d41d8cd98f00b204e9800998ecf8427e',' ',900,'en_gb',30);
-INSERT INTO users VALUES (2,'guest','Default','User','d41d8cd98f00b204e9800998ecf8427e',' ',900,'en_gb',30);
+INSERT INTO users VALUES (1,'Admin','Zabbix','Administrator','d41d8cd98f00b204e9800998ecf8427e','',900,'en_gb',30);
+INSERT INTO users VALUES (2,'guest','Default','User','d41d8cd98f00b204e9800998ecf8427e','',900,'en_gb',30);
 
 --
 -- Dumping data for table `usrgrp`
